### ISSUE TITLE
Not getting how to install the  "Site Checklist Module"

### ISSUE THREAD

**Issue by l-arnold** [pe7er/db8sitedev#58]
I have installed to component and I can operate checklists from within the component.

I cannot, however, find the  Site Checklist Module, manage it nor install it.

I have gone through both my Site and my Administrator Modules and I also cannot find anything.  I am not clear that I really need it otherthan showing it in the main Control Panel rather than going to Components.

This is all interesting.  Not quite what I need but nice work that perhaps we can add to.
thank you for your efforts.

**Comment by l-arnold** [comment]
I installed again and all was there on second install.  I actually had installed a slightly older package.

I did install a second one to my "Hathor" admin template and learned a bit about Admin Template structures.  Helpful.  Thanks again.

**Comment by pe7er** [comment]
Thanks for your comments. 

So you've managed to install it?
Did you also found the file db8-site-dev-default-checklist-en.json at https://github.com/pe7er/db8sitedev and did you manage to import it?
Thanks!

**Comment by l-arnold** [comment]
So yes,  now I did get the (default-checklist-en) imported now.  Prior there was just a barebones list.  I downloaded a Zip of your Master branch, extracted it and it was in there and I was able to import it.

I may clone/fork your repository.   I would like to do some more project work with Joomla.

I have not overwritten other files just now other than the Joomla directed Extension install.
I'll probably start with some other Json Files to see how that works for now.

Best regards and thank you!
